<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFeedback::Analyzer::MainWindow</class>
 <widget class="QMainWindow" name="UserFeedback::Analyzer::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>User Feedback Analyzer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="chartTab">
       <attribute name="title">
        <string>Charts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&amp;Chart type:</string>
            </property>
            <property name="buddy">
             <cstring>chartType</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="chartType"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QtCharts::QChartView" name="chartView">
          <property name="renderHints">
           <set>QPainter::Antialiasing|QPainter::TextAntialiasing</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="aggregationTab">
       <attribute name="title">
        <string>Aggregated Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTableView" name="aggregatedDataView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataTab">
       <attribute name="title">
        <string>Raw Data</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QTableView" name="dataView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="surveyTab">
       <attribute name="title">
        <string>Surveys</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTreeView" name="surveyView">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="uniformRowHeights">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="schemaTab">
       <attribute name="title">
        <string>Schema</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="UserFeedback::Analyzer::SchemaEditWidget" name="schemaEdit" native="true"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="UserFeedback::NotificationWidget" name="feedbackNotification" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionConnectToServer"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Products">
    <property name="title">
     <string>Pro&amp;ducts</string>
    </property>
    <addaction name="actionAddProduct"/>
    <addaction name="actionDeleteProduct"/>
   </widget>
   <widget class="QMenu" name="menu_Survery">
    <property name="title">
     <string>&amp;Survery</string>
    </property>
    <addaction name="actionAddSurvey"/>
    <addaction name="actionDeleteSurvey"/>
   </widget>
   <widget class="QMenu" name="menu_Analytics">
    <property name="title">
     <string>Anal&amp;ytics</string>
    </property>
    <widget class="QMenu" name="menu_Time_interval">
     <property name="title">
      <string>&amp;Time interval</string>
     </property>
     <addaction name="actionAggregateYear"/>
     <addaction name="actionAggregateMonth"/>
     <addaction name="actionAggregateWeek"/>
     <addaction name="actionAggregateDay"/>
    </widget>
    <addaction name="menu_Time_interval"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionContribute"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Products"/>
   <addaction name="menu_Survery"/>
   <addaction name="menu_Analytics"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="logDock">
   <property name="windowTitle">
    <string>&amp;Log Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPlainTextEdit" name="logWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="productsDock">
   <property name="windowTitle">
    <string>P&amp;roducts</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QTreeView" name="productListView">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionConnectToServer">
   <property name="text">
    <string>&amp;Connect to server...</string>
   </property>
   <property name="toolTip">
    <string>Connect to analytics server.</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit the user feedback analyzer.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::QuitRole</enum>
   </property>
  </action>
  <action name="actionAddProduct">
   <property name="text">
    <string>&amp;Add Product...</string>
   </property>
   <property name="toolTip">
    <string>Add a new product.</string>
   </property>
  </action>
  <action name="actionDeleteProduct">
   <property name="text">
    <string>&amp;Delete Product</string>
   </property>
   <property name="toolTip">
    <string>Delete product and all associated data.</string>
   </property>
  </action>
  <action name="actionAddSurvey">
   <property name="text">
    <string>&amp;Add Survey...</string>
   </property>
   <property name="toolTip">
    <string>Add new survery.</string>
   </property>
  </action>
  <action name="actionDeleteSurvey">
   <property name="text">
    <string>&amp;Delete Survey</string>
   </property>
   <property name="toolTip">
    <string>Delete a survey.</string>
   </property>
  </action>
  <action name="actionAggregateYear">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Year</string>
   </property>
   <property name="toolTip">
    <string>Aggregate data by year.</string>
   </property>
  </action>
  <action name="actionAggregateMonth">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Month</string>
   </property>
   <property name="toolTip">
    <string>Aggregate data by month.</string>
   </property>
  </action>
  <action name="actionAggregateWeek">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Week</string>
   </property>
   <property name="toolTip">
    <string>Aggregate data by week.</string>
   </property>
  </action>
  <action name="actionAggregateDay">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>&amp;Day</string>
   </property>
   <property name="toolTip">
    <string>Aggregate data by day.</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="toolTip">
    <string>Information about this program.</string>
   </property>
   <property name="menuRole">
    <enum>QAction::AboutRole</enum>
   </property>
  </action>
  <action name="actionContribute">
   <property name="text">
    <string>&amp;Contribute...</string>
   </property>
   <property name="toolTip">
    <string>Help to make this application better for you!</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QGraphicsView</extends>
   <header location="global">QChartView</header>
  </customwidget>
  <customwidget>
   <class>UserFeedback::NotificationWidget</class>
   <extends>QWidget</extends>
   <header>provider/widgets/notificationwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserFeedback::Analyzer::SchemaEditWidget</class>
   <extends>QWidget</extends>
   <header>schemaeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
